src/r-tech1/network/network.h

```diff
@@ -120,8 +120,8 @@ bool noDelay(Socket s, bool b);
 /* Whether or not new sockets re-use ports */
 void reuseSockets(bool what);
 
-void listen(Socket s);
-Socket accept(Socket s);
+void listen(Socket s) throw( NetworkException );
```

Comment:
explicitly listing exceptions in the function prototype is generally frowned upon because it causes the c++ runtime to abort if any different exceptions are thrown. exceptions are bad in general and I try to avoid them, but in this case leaving off the throw() clause is probably better.

Reply:
makes sense. replacing for a generic exception